kernel/kernel-api/src/main/java/io/delta/kernel/internal/TransactionImpl.java

```diff
@@ -437,6 +443,34 @@ private void recordTransactionReport(
     engine.getMetricsReporters().forEach(reporter -> reporter.report(transactionReport));
   }
 
+  private Optional<CRCInfo> buildPostCommitCrcInfo(
```

Comment:
add simple to method name? or some other reference that this is only if previous crc is avail